### ISSUE TITLE
Crash on android when clicking 'accept' button

### ISSUE THREAD

**Issue by freakinruben** [urbanairship/react-native-module#92]
I've integrated the `react-native-module` in my app and successfully received notifications. Everything seems to work great, except when I send a test-notification that contains buttons on android.

Clicking on the notification itself also works fine. When I click on the 'accept' or 'yes' button of the received notification, the app immediately crashes with the following exception:

```
03-14 10:22:24.643 7887-7964/? I/ReactNativeJS: 'UA notificationResponse:', { actionId: 'yes',
                                                  isForeground: true,
                                                  notification: 
                                                   { extras: 
                                                      { 'google.ttl': 2419200,
                                                        'google.sent_time': 1521019325514,
                                                        'com.urbanairship.summary': 'My summary',
                                                        'com.urbanairship.push.PUSH_ID': '...PUSH_ID...',
                                                        'com.urbanairship.push.ALERT': 'My body',
                                                        'com.urbanairship.interactive_type': 'ua_yes_no_foreground',
                                                        from: '...fromNr...',
                                                        'google.message_id': '...message_id...',
                                                        'com.urbanairship.title': 'Test message',
                                                        'com.urbanairship.push.APID': '...APID',
                                                        'com.urbanairship.push.CANONICAL_PUSH_ID': '...PUSH_ID...' },
                                                     notificationId: '1008',
                                                     title: 'Test message',
                                                     alert: 'My body' } }
03-14 10:22:24.648 550-1180/? V/WindowManager: addAppToken: AppWindowToken{29883f07 token=Token{3f22f146 ActivityRecord{28521921 u0 nl.my_app.test/nl.my_app.MainActivity t133}}} to stack=1 task=133 at 2
03-14 10:22:24.662 7887-7887/? D/Test - UALib: Analytics - New session: 2c79231f-f5a2-498e-b0a4-3a0f6fdb6c75
03-14 10:22:24.683 7887-8127/? E/AndroidRuntime: FATAL EXCEPTION: pool-28-thread-1
                                                 Process: nl.my_app.test, PID: 7887
                                                 java.lang.NullPointerException: Attempt to invoke interface method 'java.util.Set java.util.Map.entrySet()' on a null object reference
                                                     at com.urbanairship.CoreReceiver$2.run(CoreReceiver.java:335)
                                                     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
                                                     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
                                                     at java.lang.Thread.run(Thread.java:818)
```

----

Device: Nexus 4
Android: 5.1.1

**Comment by rlepinski** [comment]
Thanks for the report. We will have it fixed today.

**Comment by rlepinski** [comment]
Fixed in 1.6.0

**Comment by freakinruben** [comment]
Thanks! Works great now!
